db/seeds/scheme_ten.rb

```diff
@@ -0,0 +1,67 @@
+require 'csv'
+require_relative '../offence_code_seeder_scheme_ten.rb'
+
+FeeScheme.create(number: 9, name: 'LGFS', start_date: '2014-03-20 00:00:00')
```

Comment:
💭  Consider using `find_or_create_by` instead so that in theory this seed can be executed multiple times with no risk of creating duplicates. Also, by doing that, we can instantly assign it to a variable and avoid lines 8 and 9.